app/views/shared/_carrier_contact_information_dc.html.erb

```diff
@@ -0,0 +1,118 @@
+<div class="modal fade carrier-contact" id="<%= plan.coverage_kind %>-<%= plan.id %>" role="dialog">
```

Comment:
None of these carriers in DC or MA share the same carrier name so they should be put in the same file and then you don't need to differentiate between DC or MA.